project/AppDependencies.scala

```diff
@@ -19,12 +19,12 @@ import sbt._
 
 object AppDependencies {
 
-  private val bootstrapPlay28Version = "6.4.0"
-  private val hmrcMongoPlayVersion = "0.68.0"
+  private val bootstrapPlay28Version = "7.1.0"
+  private val hmrcMongoPlayVersion = "0.71.0"
 
   val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"                   %% "bootstrap-frontend-play-28" % bootstrapPlay28Version,
-    "uk.gov.hmrc"                   %% "play-frontend-hmrc"         % "3.8.0-play-28",
+    "uk.gov.hmrc"                   %% "play-frontend-hmrc"         % "3.23.0-play-28",
```

Comment:
Should this be 3.24.0-play-28 instead of 3.23.0-play-28?

Reply:
hmmm where did you see that version? I was based on the [following page](https://catalogue.tax.service.gov.uk/service/income-tax-pensions-frontend):

<img width="757" alt="image" src="https://user-images.githubusercontent.com/108010791/187925556-249e3f6e-d96c-4c91-b32f-2b97741285db.png">

Reply:
I see... do you know the difference between those 2? and why the discrepancy?

Reply:
Between the picture and the link to whats running where?

Reply:
As agreed in the group review, @rtna-hmrc , we'll postpone the upgrade of Scala; it's OK for this ticket to keep it at 2.12.